### PR TITLE
Yixi Chen-HW1

### PR DESCRIPTION
**Pull request by YixiC94**
Redid the exercise. A lot easier this time.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,11 @@
...
+Yixi Chen
+
+### Yixi's Bio
+
+Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:
+1. Granada, Spain
+2. Kyoto, Japan
+3. Jeruselum, Isareal  
+[Yixi's LinkedIn Page](https://www.linkedin.com/in/yixi-chen-7b42bb63/)
+![My photo at UChicago](Photo.jpg)
```
Comment (atani21) [inline README.md:11]: If you want to put the link to your LinkedIn page and your photo in different rows, I think you need to use # to start a new paragraph.

**Review by atani21 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |      x     |              |            |
| **Achievement, creativity** |     x      |              |            |
| **Ease of access**          |      x     |              |            |

Good work here. The third-person writing adds a professional touch to the whole page. Please check out my single comment regarding your photo and link, though. Judging by your code, I guess you want them to be in separate rows.

File: README.md [hunk 2]
```diff
@@ -1 +1,11 @@
-# hw01
\ No newline at end of file
+# hw01 
+Yixi Chen
+
+### Yixi's Bio
+
+Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:
+1. Granada, Spain
```
Comment (bensoltoff) [inline README.md:7]: You don't have to number the list items. Markdown will do that for you.

```
1. Granada, Spain
1. Kyoto, Japan
1. Jeruselum, Isareal  
```

1. Granada, Spain
1. Kyoto, Japan
1. Jeruselum, Isareal

File: README.md (hunk 1 above)
Comment (bensoltoff) [reply README.md:11]: Actually you just need a line break

```
Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:

1. Granada, Spain
2. Kyoto, Japan
3. Jeruselum, Isareal  

[Yixi's LinkedIn Page](https://www.linkedin.com/in/yixi-chen-7b42bb63/)

![My photo at UChicago](Photo.jpg)
```

Yixi is originally from China. She graduated from **Mount Holyoke College** with double major in **History and Economics**. After graduation, she worked at Mastercard as an analyst for three years. Her top 3 travel destinations are:

1. Granada, Spain
2. Kyoto, Japan
3. Jeruselum, Isareal  

[Yixi's LinkedIn Page](https://www.linkedin.com/in/yixi-chen-7b42bb63/)

My photo at UChicago

File: README.md [hunk 3]
```diff
@@ -1 +1,11 @@
...
+2. Kyoto, Japan
+3. Jeruselum, Isareal
```
Comment (bensoltoff) [inline README.md:9]: Typo in Israel. RStudio actually has a built-in spell checker you can use to proof-read Markdown documents

**Review by bensoltoff — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |           |   x           |            |
| **Presentation**            |           |     x         |            |
| **Achievement, creativity** |           |      x        |            |
| **Ease of access**          |           |     x         |            |

* Your commit messages could use a bit more description as to what you actually did

**Review by adamberger20 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |           |      x        |            |
| **Coding strategy**         |     x      |              |            |
| **Presentation**            |     x      |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |     x      |              |            |

Remarks:

* Cool bio! I feel like I learned a lot about you in a short space, and the third-person writing was a nice touch.
* All the features turned out cleanly: headers, bold text, lists, photos, and the link to your LinkedIn page.
* I think it could have benefitted from a bit more information. Tell us about the travel destinations!
* Nice work!

**Pull request closed without merging**